core/integration/main_test.go

```diff
@@ -462,6 +462,14 @@ func InitializeScenario(s *godog.ScenarioContext) {
 		execsetup.assetsEngine.SetPermissive()
 		return nil
 	})
+
+	s.Step(`^the parties should have the following position changes for market "([^)]+)":$`, func(mkt string, table *godog.Table) error {
+		return steps.PartiesShouldHaveTheFollowingPositionStatus(execsetup.broker, mkt, table)
+	})
+
+	s.Step(`^the parties should have the following aggregated position changes for market "([^)]+)":$`, func(mkt string, table *godog.Table) error {
```

Comment:
Do we really need this if it's not been used even in this PR?

Reply:
We don't use it in this PR, indeed. I added these steps before the position status was added to the PnL table. I don't think we need to include the position status as a separate step per se, but the aggregated position change step might be useful in future. It allows us to check all states a party passes through in a more complex scenario (e.g. -> party is distressed, but can't be closed out, price changes increase their balance so they are no longer distressed, price changes again, and the party gets closed out). With this step we can ensure the party passes through all the expected states in a single step/check. For that reason, I'd quite like to keep this one.

It's also worth keeping in mind that core, currently, still has the positions plugin in the code, solely for use in the integration tests. Having these steps checking position status specifically might be worth keeping, because the positions plugin is increasingly something we can classify as legacy code.

Reply:
Fair enough, thanks for the explanation.